src/internal/methods/NeoBits.h

```diff
@@ -5,7 +5,7 @@ NeoPixel library helper functions for Methods.
 Written by Michael C. Miller.
 
 I invest time and resources providing this open source code,
-please support me by donating (see https://github.com/Makuna/NeoPixelBus)
+please support me by dontating (see https://github.com/Makuna/NeoPixelBus)
```

Comment:
You are introducing a typo here. 😉

Reply:
Pretty sure that Typo has been there for a while. It's in the comments hardly relevant.